docs/api-testing/schedule-test.md

```diff
@@ -12,35 +12,29 @@ The **Schedule** tool, accessible from within each test, allows you to schedule
 ## What You'll Need
 
 * A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up)).
-* An existing API Testing Project and Test. For details on how to create them, see the [Quickstart](/api-testing/quickstart/) guide.
+* An existing, published API Test. For details on how to create one, see the [Quickstart](/api-testing/quickstart/).
 
 
-## Publish the Working Copy
+## Creating a New Test Schedule
 
-Our API Testing interface has a unique _Working Copy_ / _Published Copy_ system that allows you to edit a test without affecting the live, currently active version.
+:::caution Required Step
+Do not proceed until you've [published the working copy of your test](/api-testing/quickstart/#publish-your-test).
```

Comment:
@LoryApiFortress So I should reword that sentence to something like this?

:::caution Required Step
Do not proceed until you've [published your test](/api-testing/quickstart/#publish-your-test).
:::

Reply:
@kimsauce sorry, maybe I misunderstood your phrase. my doubt is because, if the test is not been published at least one time, the schedule tab is not available at all so user cannot schedule anything